fuzzers/fuzzbench/fuzzbench_ctx/Cargo.toml

```diff
@@ -26,8 +29,12 @@ which = "6.0"
 
 [dependencies]
 libafl = { path = "../../../libafl/" }
-libafl_bolts = { path = "../../../libafl_bolts/" }
-libafl_targets = { path = "../../../libafl_targets/", features = ["sancov_pcguard_hitcounts", "sancov_cmplog", "libfuzzer", "sancov_ctx"] }
+libafl_targets = { path = "../../../libafl_targets/", features = [
```

Comment:
is there some sort of autoformat for TOML files we can add to CI/scripts?

Reply:
This is the underlying library that the VSCode TOML plugin is using:

https://taplo.tamasfe.dev/cli/usage/formatting.html

I don't think it's worth it to have it be checked in CI though. Maybe in a hypothetical future where formatting fixes are automatically applied to PRs via CI?

Reply:
> formatting fixes are automatically applied in CI

No, this is actually dangerous. There are a few cases where I've had both TOML and Rust formatters completely delete code. It should just be checked in CI.

Reply:
Checking could be cool though